src/ui/features/onboarding/Success/WebAppMessageHandler.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useRef } from 'react';
+import { invariant } from 'src/shared/invariant';
+import { isObj } from 'src/shared/isObj';
+
+const ZERION_WEB_APP_URL = new URL(
+  process.env.NODE_ENV === 'development'
+    ? 'http://localhost:3000'
+    : 'https://app.zerion.io'
+);
+
+enum WebAppCallbackMethod {
```

Comment:
Aren't enums considered deprecated?

Reply:
Ah, right, changed to just `type` ✅

Reply:
Are they?...

Reply:
Not really, some consider them bad, some don't... I think in this context its not a big deal: I just wanted to have some "sum"-type to enumerate possible callback methods